HISTORY.md

```diff
@@ -4,6 +4,7 @@
 * Remove HDFS support from main repo.
 * Remove librados support from main repo.
 * Remove deprecated API DB::AddFile from main repo.
+* Remove deprecated API AdvancedColumnFamilyOptions::soft_rate_limit
```

Comment:
Thanks for being the first to drop "from main repo".

Reply:
let's see who merge it first https://github.com/facebook/rocksdb/pull/9454